Magboots for Sangheili Ranger, Kig-Yar Ranger and Jiralhanae EVA

5crownik007 opened the pull request:
🆑 5crownik007
bugfix: Sangheili Ranger boots now act as magboots. Sangheili Ranger helm now has major flash protection.
rscadd: Kig-Yar Rangers and Jiralhanae EVA now have boots of their own(magboots)
bugfix: Marine EVA boots icon no longer breaks when state is toggled.
/🆑

5crownik007 committed:
Ranger armour changes

Ranger boots are now also magboots, so you can actually not fall over
Also, the ranger helmet now has FLASH_PROTECTION_MAJOR

5crownik007 committed:
marine evaboots fix

no longer disappear when switching states between on and off, or putting a knife in

5crownik007 committed:
sang rangerboots

Switching the obj path for ranger boots

5crownik007 committed:
Jiralhanae EVA boots

jiralahane eva now has magboots

5crownik007 committed:
kigyar eva boots

kigyar ranger now has magboots

5crownik007 committed:
magboots for jiral eva and kig ranger

adds magboots for jiral eva and kig ranger into their vendors

5crownik007 committed:
adds magboots + uniform

gives magboots + uniform to kigyar ranger outfit

X0-11 commented on code/modules/halo/covenant/species/kigyar/kigyar_ranger.dm:
```diff
@@ -22,6 +22,23 @@
+	heat_protection = LEGS|FEET
+	min_cold_protection_temperature = SPACE_SUIT_MIN_COLD_PROTECTION_TEMPERATURE
+	max_heat_protection_temperature = FIRESUIT_MAX_HEAT_PROTECTION_TEMPERATURE
+	armor_thickness = 15
```
keep at 20, no need to lower this

5crownik007 replied:
it's 15 because all the other kig ranger stuff is 15

X0-11 replied:
*why*

5crownik007 replied:
I don't know, I didn't program the other kig ranger equipment, I just followed the example

caelaislinn commented on code/modules/halo/clothing/marine.dm:
```diff
@@ -359,6 +359,7 @@
 	icon_override = MARINE_OVERRIDE
 	item_state = "evalegs"
 	icon_state = "evalegs"
+	icon_base = null
```
Did you test this?

5crownik007 replied:
Yes, it works

caelaislinn commented on code/modules/halo/covenant/species/sangheili/clothing.dm:
```diff
@@ -392,16 +392,34 @@
+	item_flags = STOPPRESSUREDAMAGE|THICKMATERIAL|AIRTIGHT
+	body_parts_covered = LEGS|FEET
+	cold_protection = LEGS|FEET
+	min_cold_protection_temperature = SPACE_SUIT_MIN_COLD_PROTECTION_TEMPERATURE*/
```
This can be just deleted, the magboot code above makes this all redundant

5crownik007 replied:
Removed

5crownik007 committed:
Merge branch 'alpha' into Rangerchanges

5crownik007 committed:
deletes redundant ranger boot code

Old ranger boots are no longer needed since we have the proper magboot ones now
